openc3-cosmos-init/plugins/packages/openc3-tool-common/src/components/widgets/ValueWidget.vue

```diff
@@ -99,13 +99,20 @@ export default {
     },
   },
   created() {
-    this.verifyNumParams('VALUE', 3, 3, 'VALUE <TARGET> <PACKET> <ITEM>') // TYPE, WIDTH
-    // Note: TYPE is parameters[3]
-    this.setWidth(
-      parseInt(this.parameters[4]) + INPUT_PADDING,
-      'ch',
-      this.width
```

Comment:
How did this ever work? We had items with more than 3 params? What in the other PR broke this?

Reply:
The max num parameters set as 3 was ignored before my sonarqube bugfix PR.   
So things with more than 3 parameters were fine.  Suddenly they weren't when that code actually works.   Turns out up to six parameters were getting passed to the ValueWidget.